index.bs

```diff
@@ -1140,7 +1140,7 @@ nothrow>ReadableStreamAddReadIntoRequest ( <var>stream</var> )</h4>
   The case where <var>stream</var>.\[[state]] is <code>"closed"</code>, but <var>stream</var>.\[[closeRequested]] is
   <emu-val>false</emu-val>, will happen if the stream was closed without its controller's close method ever being
   called: i.e., if the stream was closed by a call to {{ReadableStream/cancel(reason)}}. In this case we allow the
-  controller's <code>close</code> method to be called and silently do nothing, since the cancelation was outside the
+  controller's <code>close</code> method to be called and silently do nothing, since the cancellation was outside the
```

Comment:
I think "cancelation" is wrong too, but WhatWG style says that's how we have to spell it: https://wiki.whatwg.org/wiki/Style